packages/playground/src/components/app-home/app-home.tsx

```diff
@@ -0,0 +1,17 @@
+import { Component } from "@stencil/core";
+
+@Component({
+  tag: "app-home",
+  styleUrl: "app-home.css",
+  shadow: true
+})
+export class AppHome {
+  render() {
+    // tslint:disable-next-line:prettier
```

Comment:
Why?

Reply:
At first, because it forced me to use a self-closing tag for an empty DIV, which isn't correct AFAIK.